content/technology/Material UI/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Material UI
+description: Material UI is one of the popular react UI framework used in Altinn Studio/ Altinn Studio Apps
+weight: 100
+tags: ["tjenester 3.0", "MAterial UI"]
+---
+
+#### Material UI 
+Material UI is used in Altinn Studio as the React UI framework. MAterial UI was chosen because
+- It is based on the Google's material design. 
+- It focuses exclusively on the React library
```

Comment:
Not sure which React library you're referring to.
It's a library that uses React to render components with the correct css.

Reply:
i mean it doesn't focus on other UI libraries. But now i changed this to react components